tensorboard/pip_package/build_pip_package.sh

```diff
@@ -142,6 +142,7 @@ smoke() {
   if [ -n "${smoke_tf}" ]; then
     pip install -qU "${smoke_tf}"
   fi
+  pip uninstall -qy tensorboard tb-nightly  # Drop any conflicting packages
```

Comment:
You might as well do this one line up, inside the case where we install the TF package?  Otherwise this is a fresh virtualenv so there's no need to do the uninstall.

Reply:
Sure; fine with me.